packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -409,14 +409,14 @@ export class DocumentDeltaConnection
 
                 // This is not an socket.io error - it's Fluid protocol error.
                 // In this case fail connection and indicate that we were unable to create connection
-                fail(false, this.createErrorObject("connect_document_error", error));
+                fail(false, this.createErrorObject("connectDocumentError", error));
             }));
 
             this.socket.emit("connect_document", connectMessage);
 
             // Give extra 2 seconds for handshake on top of socket connection timeout
             this.socketConnectionTimeout = setTimeout(() => {
-                fail(false, this.createErrorObject("Timeout waiting for handshake from ordering service"));
+                fail(false, this.createErrorObject("timeoutWaitingForHandshakeFromOrderingService"));
```

Comment:
name is too long, consider shortening it.
Also, should we use strings instead and use tagger to give them IDs?
These strings will add up in bundle size, so I think we definitely should look into it.
Can you please open an issue to access it?

Reply:
I can open a separate issue for that. This PR will not increase the bundle size as I am not lengthening the message.
Will think about some short fluidErrorCode here but I do think it should be somewhat informative.

Reply:
Opened an issue here: https://github.com/microsoft/FluidFramework/issues/7465